### DISCUSSION TITLE
How to skip out of range notes in learning mode on 61 keys keyboard?

### DISCUSSION THREAD

**Discussion by artisdom** [onlaj/Piano-LED-Visualizer#516] [Q&A]
My keyboard has only 61 keys, with songs that have out of range notes for my keyboard, how to skip these notes in learning mode, currently it seems stuck and keep wait for me to press the notes, but they are out of range for my keyboard.

Any ideas, thanks a lot.

**Comment by onlaj** [comment]
Hey, that's a good point. I'm not sure if it's a bug or feature request, so I've added both labels :D 
I will try to add support for that in future updates.
